rfqmsg/reject.go

```diff
@@ -230,9 +205,9 @@ func NewQuoteRejectFromWireMsg(wireMsg WireMessage) (*Reject, error) {
 	}
 
 	// Ensure that the message version is supported.
-	if msgData.Version > latestRejectVersion {
+	if msgData.Version.Val > latestRejectVersion {
```

Comment:
with this update I think that we can handle precisely the latest version (bumped in this PR) and no other. Including no earlier version. So Should this be `!=` rather than `>`?